wasi-filesystem.html

```diff
@@ -1,5 +1,260 @@
+<h1>Import interface <code>wasi-poll</code></h1>
```

Comment:
I assume this file is automatically generated from the `.md`?   What is the process by which that happens?  What is reason to check it into the git repo?

Reply:
Yes, it's auto-generated from the .wit.md files. There's no super urgent reason to check it in; I was just thinking people might find it useful, or maybe it might help inspire someone to improve the HTML or contribute a way to auto-publish the HTML somewhere easily viewable. I'm open to ideas here though.

Reply:
So for most proposals we are just checking in `.md` for now?  And this is just an experiment.  Seems find if thats that case. 

Is the github rendering of the `.md` not considered long term good enough?   (I can't seem to see the rendering of the checked-in html itself via github, so I can't tell if it looks very different).

Reply:
Most proposals should start with the [wasi proposal template](https://github.com/WebAssembly/wasi-proposal-template/) which will set up the generated .md and .html files. That's what wasi-filesystem here is now following. If people have ideas for improving the proposal repo structure, the proposal template is the place to have them!